src/types/task.list.ts

```diff
@@ -0,0 +1,52 @@
+import { DateTime } from "luxon";
+import { readableFormat } from "../utils/date.formatter";
+
+export interface TaskList {
```

Comment:
A lot of these look they they come from the prototype. for our template we'll just need indicators/flags to toggle the status of each task. Looking at the story there only seems to be 5 or 6 tasks on the screen.

Reply:
7 tasks for LTD companies and 4 for LLPs.

Reply:
you could create an enumerated type for the different states and pass that in to the template

Reply:
These really are just placeholders so that we can display the page there will be changes around this page as we develop this further.